src/sets.jl

```diff
@@ -1534,6 +1534,24 @@ Base.copy(set::Table) = Table(copy(set.table))
 
 Base.:(==)(x::Table{T}, y::Table{T}) where {T} = x.table == y.table
 
+"""
+    Reified(set::AbstractSet)
+
+The constraint ``[z; f(x)] \\in Reified(S)`` ensures that ``f(x) \\in S`` if and
```

Comment:
Why `f(x)`? I would simply use `x`, with `x \\in S`.

Reply:
We've been using this notation to mean that it also supports `VectorAffineFunction` etc, not just `VectorOfVariables`.